tailwind.config.js

```diff
@@ -85,6 +85,13 @@ module.exports = {
         'slide-in-from-top':
           'slide-in-from-top 300ms cubic-bezier(.41,.73,.51,1.02)',
         'shrink-and-fade-out': 'shrink-and-fade-out 150ms ease-in forwards',
+        slideDownAndFade:
+          'slideDownAndFade 400ms cubic-bezier(0.16, 1, 0.3, 1)',
+        slideLeftAndFade:
+          'slideLeftAndFade 400ms cubic-bezier(0.16, 1, 0.3, 1)',
+        slideUpAndFade: 'slideUpAndFade 400ms cubic-bezier(0.16, 1, 0.3, 1)',
+        slideRightAndFade:
+          'slideRightAndFade 400ms cubic-bezier(0.16, 1, 0.3, 1)',
```

Comment:
We’re currently mixing naming styles for animations here, such as 'slide-in-from-top' and slideDownAndFade, which can lead to inconsistency. Additionally, since these animations are specific to this component, it might be better to move them to a separate CSS file. This would help keep the Tailwind config file more manageable, especially if we continue adding component-specific animations.

Reply:
Move this to a separate CSS file

Reply:
WIP

Reply:
I tried moving these animations outside the Tailwind config, but they stopped being imported when I did, so the animation stopped working. I'm not sure as to why at this time.